doc/overview.rst

```diff
@@ -73,6 +73,90 @@ produced.
+      |                |                     | Relativistic              | :cpp:class:`celeritas::RelativisticBremModel` |   1 GeV - 100 TeV        |
+      |                +---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |                | Coulomb scattering  | Urban                     | :cpp:class:`celeritas::UrbanMsc` [1]_         |   10 eV - 100 MeV        |
+      |                +---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |                | Annihilation        |:math:`e^+-e^- \to 2\gamma`| :cpp:class:`celeritas::EPlusGGModel`          |       0 - 100 TeV        |
+      +----------------+---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |:math:`\gamma`  | Photoelectric       | Livermore                 | :cpp:class:`celeritas::LivermorePEModel`      |       0 - 100 TeV        |
+      |                +---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |                | Compton scattering  | Klein - Nishina           | :cpp:class:`celeritas::KleinNishinaModel`     |       0 - 100 TeV        |
+      |                +---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |                | Pair production     | Bethe - Heitler           | :cpp:class:`celeritas::BetheHeitlerModel`     |       0 - 100 TeV        |
+      |                +---------------------+---------------------------+-----------------------------------------------+--------------------------+
+      |                | Rayleigh scattering | Livermore                 | :cpp:class:`celeritas::RayleighModel`         |       0 - 100 TeV        |
+      +----------------+---------------------+---------------------------+-----------------------------------------------+--------------------------+
+
+  .. [1] Multiple Scattering using the Urban Model is only applied up to 100MeV in Celeritas, with no model used above this energy.  
+
+.. only:: latex
+
+   .. raw:: latex
+      
+      \begin{table}[h]
+        \caption{Electromagnetic physics processes and models available in Celeritas.}
+        \begin{threeparttable}
+        \begin{tabular} {|l | l | l | l | r |}
+          \hline
+          \textbf{Particle}         & \textbf{Processes}              & \textbf{Models}      & \textbf{Celeritas Implementation}                     & \textbf{Applicability} \\
+          \hline
+          \multirow{4}{*}{$e^-$}    & Ionisation                      & Moller               & \texttt{\scriptsize celeritas::MollerBhabhaModel}     & 0 - 100 TeV \\
```

Comment:
Pardon my pedantry but all the numeric ranges in this table should have en dashes 😉 , and Møller needs a slashed o:
```suggestion
          \multirow{4}{*}{$e^-$}    & Ionisation                      & Møller               & \texttt{\scriptsize celeritas::MollerBhabhaModel}     & 0--100 TeV \\
```